### ISSUE TITLE
[Accessibility]: Search contexts management ARC `PageHeader` Issue

### ISSUE THREAD

**Issue by gitstart-sourcegraph** [sourcegraph/sourcegraph#35613]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34191

### Problem description

The following ARC toolkit issues exist in the [search context management pages](https://sourcegraph.com/contexts): 

- [ ] Headers should not be H1, should be H2 styled as H1, on all pages ([list page](https://sourcegraph.com/contexts), [view page](https://sourcegraph.com/contexts/global), and [add/edit page](https://sourcegraph.com/contexts/new))
![CleanShot 2022-04-29 at 16 47 04](https://user-images.githubusercontent.com/206864/166081367-c4d85f36-e771-4f00-a17d-d77b0fa905fd.png)
![CleanShot 2022-04-29 at 16 47 22](https://user-images.githubusercontent.com/206864/166081377-06ac719f-8fc5-4c8b-929c-30dc1c01ecf2.png)
![image](https://user-images.githubusercontent.com/206864/166081393-280eb22c-3128-43fa-be80-06c425718e5c.png)

## Ref 
https://github.com/sourcegraph/sourcegraph/issues/34741

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by gitstart-sourcegraph** [comment]
Done in [this PR](https://github.com/sourcegraph/sourcegraph/pull/35776)
